tests/click_tests/device_menu/test_wipe_code.py

```diff
@@ -0,0 +1,312 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2025 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import re
+from typing import TYPE_CHECKING
+
+import pytest
+
+from ... import translations as TR
+from ..common import go_next
+from ..test_pin import (
+    PIN1,
+    PIN4,
+    _cancel_pin,
+    _delete_pin,
+    _enter_two_times,
+    _input_pin,
+    _input_see_confirm,
+    _see_pin,
+)
+from .common import (
+    Menu,
+    assert_device_screen,
+    close_device_menu,
+    enter_pin,
+    open_device_menu,
+)
+from .test_pin import Situation, prepare_pin_dialogue
+
+if TYPE_CHECKING:
+    from trezorlib.debuglink import DebugLink
+
+    from ...device_handler import BackgroundDeviceHandler
+# Trezor Safe 7 only
+pytestmark = [pytest.mark.models("eckhart")]
+
+WIPE_CODE = "2"
+WIPE_CODE_TITLE = "wipe_code__title"
```

Comment:
nit (to avoid `TR.translate(WIPE_CODE_TITLE)` calls below):
```suggestion
WIPE_CODE_TITLE = TR.wipe_code__title
```
(OTOH, we can probably inline it - right?)

Reply:
There was an issue when using foreign languages, so I had to use `TR.translate(WIPE_CODE_TITLE)` calls.